src/frontend/posts_page/components/InputBlock.tsx

```diff
@@ -1,18 +1,15 @@
 import React from "react";
-import SetPublic from './SetPublic';
+import VisibilitySetting from './VisibilitySetting';
 import Select from 'react-select';
-import SetTextEditor from './SetTextEditor';
+import TextEditorSetting from './TextEditorSetting';
 
-interface TextareaProps {
-}
-
-interface TextareaState {
+interface IInputBlockState {
     editing: boolean,
     message: string,
 }
 
-class Textarea extends React.Component<TextareaProps, TextareaState> {
-    constructor(props: TextareaProps) {
+class InputBlock extends React.Component<{}, IInputBlockState> {
```

Comment:
Reason for renaming: This class contains multiple text areas and other components, so I think `Textarea` is not a suitable name for this class.